File: Java/EstruturasDeControl.java
// Ya vimos un tipo de estucturas de control que son bucles 
// ahora vamos a ver las las estururas condicionales
/* Las estructuras condionales se utilizan para realizar decisiones en el flujo del programa*/
public class EstruturasDeControl {
    public static void main(String[] args) {
        int edad = 17;
        /* Estructura Condicional If */
        if (edad >= 18){
            System.out.println("Eres mayor de edad");
            } else{
                System.out.println("Eres menor de edad");
             }
                /* Estructura Else If o Else If */
                /* Si la primera condición es falsa, entonces mira si la siguiente es verdadera*/
                if (edad == 18){
                    System.out.println("Eres de legal para votar");
                    }else if (edad > 18 && edad <=25 ){
                        System.out.println("Eres miembro activo de la fuerza laboral");
                    }
                /* Estructura Switch Case */
                String colorFavorito = "azul";
                switch (colorFavorito) {
                    case "rojo":System.out.println("Tu color favorito es rojo");
                    break;
                    case "verde":System.out.println("Tu color favorito es verde");
                    break;
                    default:System.out.println("Tu color favorito no es azul ni rojo ni el verde");
                }
        }           
}

File: PHP/EstruturasDeControl.php
<?php
  // Estructuras condicionales
  //Estructura if:
  $edad = 20;
  if ($edad >= 18) {
      echo "Eres mayor de edad. \n";
  }
  echo "<br>";
  //Estructura if y else:
    if ($edad >= 18) {
        echo "Eres mayor de edad.\n";
    } else {
        echo "Eres menor de edad. \n";
    }
    echo "<br>";
  // Estructura if, else if y else:
  $nota = 75;
  if ($nota >= 90) {
      echo "Tienes una A \n";
  } elseif ($nota >= 80) {
      echo "Tienes una B \n";
  } elseif ($nota >= 70) {
      echo "Tienes una C \n";
  } else {
      echo "Tienes una nota inferior a C \n";
  }
  echo "<br>";
  // switch
  $opcion = 2;
  switch ($opcion) {
      case 1:
          echo "Seleccionaste la opción 1";
          break;
      case 2:
          echo "Seleccionaste la opción 2";
          break;
      case 3:
          echo "Seleccionaste la opción 3";
          break;
      default:
          echo "Opción no válida";
          break;
  }
?>